### PR TITLE
cleanup: missed CMO code/definitions

### PR DESCRIPTION
**Pull request by joez-f**
### Change description ###

More CMO cleanup that was missed. Also cleaned up model import

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/enums/DirectionAssignee.java [hunk 1]
```diff
@@ -18,14 +18,6 @@ public String getValue() {
         return value;
     }
 
-    public String toCustomDirectionField() {
```
Comment (lewisbirks) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/enums/DirectionAssignee.java:21]: Do we still need this one for SDO?

Comment (joez-f) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/enums/DirectionAssignee.java:21]: It had 0 uses so I assume not

**Review by lewisbirks — CHANGES_REQUESTED** [review]
`linkToAttachedDocument` in `service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/CaseManagementOrderEmailContentProvider.java` is no longer used

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/CaseManagementOrderEmailContentProvider.java [hunk 2]
```diff
@@ -1,48 +1,25 @@
...
 
 import static org.apache.commons.lang3.StringUtils.uncapitalize;
 import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.DIGITAL_SERVICE;
-import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.buildSubjectLineWithHearingBookingDateSuffix;
-import static uk.gov.hmcts.reform.fpl.utils.NotifyAttachedDocumentLinkHelper.generateAttachedDocumentLink;
 import static uk.gov.hmcts.reform.fpl.utils.PeopleInCaseHelper.getFirstRespondentLastName;
 
 @Slf4j
 @Service
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
```
Comment (lewisbirks) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/CaseManagementOrderEmailContentProvider.java:20]: This line is no longer required I think

Comment (lewisbirks) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/CaseManagementOrderEmailContentProvider.java:20]: I don't think the logging annotation is required either

File: service/src/main/java/uk/gov/hmcts/reform/fpl/NotifyTemplates.java [hunk 3]
```diff
@@ -26,8 +26,6 @@ private NotifyTemplates() {
         "90c5f4b3-738b-4c8d-b433-c8c90095f815";
     public static final String CMO_READY_FOR_JUDGE_REVIEW_NOTIFICATION_TEMPLATE_JUDGE =
         "5a092766-6a39-4c00-a7a7-a31a72c60b48";
-    public static final String CMO_READY_FOR_PARTY_REVIEW_NOTIFICATION_TEMPLATE =
```
Comment (lewisbirks) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/NotifyTemplates.java:29]: You going to deprecate the notification template in gov notify?

**Review by lewisbirks — COMMENTED** [review]
Looks good, couple of minor things

**Review by lewisbirks — APPROVED** [review]

**Pull request merged**